packages/ebayui-core-react/src/ebay-flag/types.ts

```diff
@@ -0,0 +1,270 @@
+/* This file is autogenerated, do not edit - it will be overwritten anyway */
+
+export type Icon =
```

Comment:
Would be good to have this in a common place so both marko and react can use it.
Anyway can do this in a follow up.